File: WhichPharmaPortal/WhichPharmaPortal/ClientApp/src/models/RFQQuote.ts
import { RFQQuoteState } from './RFQQuoteState';

export interface RFQQuoteV2 {
    data: {
        id: string;
        rfqNr: string;
        name: string;
        state: RFQQuoteState;
        endingDate?: string;
        rfqDescription?: string;
        activeSubstances?: string;
        expDate?: string;
        leadTimeToDeliver?: string;
        availabilityPacks?: number;
        exwNetPriceEuro?: string;
        unitQuant?: string;
        packSize?: string;
        supplierReplyForm?: string;
        productCode?: string;
        countryOfOrigin?: string;
        maHolder?: string;
        comments?: string;
    };
}

export interface RFQQuote {
    id: string;
    rfqNr: string;
    name: string;
    state: RFQQuoteState;
    endingDate?: string;
    lastUpdateDate?: string;
    rfqDescription?: string;
    activeSubstances?: string;
    expDate?: string;
    leadTimeToDeliver?: string;
    availabilityPacks?: string | number;
    exwNetPriceEuro?: string;
    currency?: string;
    priceInCurrency?: string;
    updatedBy?: string;
    createdBy?: string;
    unitQuant?: string;
    numOfPacks?: string;
    packSize?: string;
    productCode?: string;
    countryOfOrigin?: string;
    maHolder?: string;
    comments?: string;
    isAlternative?: boolean;
    // attachment?: string;
}

export interface IFilterMembers {
    key: string;
    checked: boolean;
}

File: WhichPharmaPortal/WhichPharma.Models/Client/RFQQuote.cs
using System;
using WhichPharma.Models.Attributes;

namespace WhichPharmaPortal.Models.Client
{
    public class RFQQuote
    {
        public string Id { get; set; }
        public string RfqNr { get; set; }
        public string Name { get; set; }
        public RFQQuoteState State { get; set; }
        [Optional]
        public DateTime EndingDate { get; set; }
        [Optional]
        public string RfqDescription { get; set; }
        [Optional]
        public string ActiveSubstances { get; set; }
        [Optional]
        public string ExpDate { get; set; }
        [Optional]
        public string LeadTimeToDeliver { get; set; }
        [Optional]
        public string AvailabilityPacks { get; set; }
        [Optional]
        public int? Availability { get; set; }
        [Optional]
        public string ExwNetPriceEuro { get; set; }
        [Optional]
        public string PriceCurrencyToEuro { get; set; }
        [Optional]
        public string UnitQuant { get; set; }
        [Optional]
        public string PackSize { get; set; }
        [Optional]
        public string ProductCode { get; set; }
        [Optional]
        public string Currency { get; set; }
        [Optional]
        public string CountryOfOrigin { get; set; }
        [Optional]
        public string MAHolder { get; set; }
        [Optional]
        public string Comments { get; set; }
        [Optional]
        public bool IsAlternative { get; set; }


        //[Optional]
        //public string Attachment { get; set; }
    }
}
